report.go

```diff
@@ -58,6 +71,11 @@ func writeError(w io.Writer, err error, p reportParams) {
 		var v string
 		if u, ok := value.(Unquoted); ok {
 			v = string(u)
+		} else if s, ok := value.(SuppressedIfLong); ok {
+			v = p.format(s.Value)
+			if !testingVerbose() && strings.Count(v, "\n") >= longValueLines {
+				v = "<suppressed because too long, use -v for the full output>"
+			}
```

Comment:
"because too long" reads a bit awkwardly. Also, maybe it might be nice to say just how long it is.

Something like this maybe?

```suggestion
			if !testingVerbose() {
				if n := strings.Count(v, "\n"); n > longValueLines {
					v = fmt.Sprintf("<suppressed due to length (%d lines omitted)>", n)
				}
			}
```

Reply:
Done.